treeherder/perf/management/commands/backfill_perf_jobs.py

```diff
@@ -0,0 +1,27 @@
+from django.core.management.base import BaseCommand
+
+from treeherder.perf.investigation_aids import (BackfillTool,
+                                                TaskclusterModel)
+
+
+class Command(BaseCommand):
```

Comment:
Even though this command will not be used in the PerfSheriffBot I think we should keep it to have direct access to the BackfillTool logic outside of the bot